obelisk/cpp/zoo/robot/leap_hand_interface.h

```diff
@@ -0,0 +1,126 @@
+#include <cmath>
+#include <iostream>
+
+#include "dynamixel_sdk.h"
+#include "obelisk_robot.h"
+#include "obelisk_ros_utils.h"
+#include "obelisk_sensor_msgs/msg/joint_encoders.h"
+
+namespace obelisk {
+    using leap_control_msg = obelisk_control_msgs::msg::PositionSetpoint;
+    using leap_sensor_msg  = obelisk_sensor_msgs::msg::JointEncoders;
+
+    /**
+     * @brief Obelisk robot interface for the Leap hand hardware
+     */
+    class ObeliskLeapHand : public ObeliskRobot<leap_control_msg> {
+
+      public:
+        /**
+         * @brief Construct a new Obelisk Leap Hand object
+         *
+         * @param node_name Name of the node
+         * @param state_timer_key Key for the state timer
+         * @param state_pub_key Key for the state publisher
+         */
+        ObeliskLeapHand(const std::string& node_name, const std::string& state_timer_key = "timer_sensor",
+                        const std::string& state_pub_key = "pub_sensor")
+            : ObeliskRobot<leap_control_msg>(node_name),
+              port_handler_(dynamixel::PortHandler::getPortHandler("/dev/ttyUSB0")),
+              packet_handler_(dynamixel::PacketHandler::getPacketHandler()),
+              group_reader_(port_handler_, packet_handler_), group_writer_(port_handler_, packet_handler_),
+              state_timer_key_(state_timer_key), state_pub_key_(state_pub_key) {
+            this->RegisterObkTimer("timer_sensor_setting", state_timer_key_,
+                                   std::bind(&ObeliskLeapHand::PublishState, this));
+            this->template RegisterObkPublisher<leap_sensor_msg>("pub_sensor_setting", state_pub_key_);
+            if (port_handler_->openPort()) {
+                std::cout << "Opened port\n";
+            } else {
+                std::cerr << "Failed to open port\n";
+            }
+            if (port_handler_->setBaudRate(BAUDRATE)) {
+                std::cout << "Set baud rate to" << BAUDRATE << "\n";
+            } else {
+                std::cerr << "Failed to set baud rate\n";
+            }
+            for (int i = 0; i < N_MOTORS; i++) {
+                int err = packet_handler_->write1ByteTxRx(port_handler_, i, 64, 1); // Enable torque
+                err |= packet_handler_->write2ByteTxRx(port_handler_, i, 84, 500);  // kP
+                err |= packet_handler_->write2ByteTxRx(port_handler_, i, 82, 10);   // kI
+                err |= packet_handler_->write2ByteTxRx(port_handler_, i, 80, 50);   // kD
```

Comment:
can we set these numbers to variables instead, so the semantic meaning is clear?

For example, `torque_enable = 1;` or `kP = 500;` would go a long way in terms of readability

Reply:
also, i'm assuming the numbers like 64, 84, etc. are port numbers. Can you leave a comment demonstrating how you know those are the correct numbers for future reference? Otherwise, it's too much of a magic number.